website/docs/tutorials/basic/running_your_app/2_multirun.md

```diff
@@ -21,6 +25,29 @@ The following sweeps over all combinations of the dbs and schemas.
 ```
 The printed configurations have been omitted for brevity.
 
+You can also define sweeping in the input configs by overriding
+`hydra.sweeper.params`. Using the above example, the same multirun could be achieved via the following config.
+
+```yaml
+hydra:
+  sweeper:
+    params:
+      db: mysql,postgresql
+      schema: warehouse,support,school
+```
+
+The syntax are consistent for both input configs and commandline overrides.
+If a sweep is specified in both an input config and at the command line,
+then the commandline sweep will take precedence over the sweep defined 
+in the input config. If we run the same application with the above input config and a new commandline override:
```

Comment:
Since the syntax is not yet standardized across all of our supported sweepers, I think this could be confusing to a user who e.g. wants to use the nevergrad sweeper and finds that `hydra.sweeper.params` is not supported.

Reply:
(Then again, these docs makes sense for the next release assuming that all the sweepers will be standardized)

Reply:
I've added an info section to highlight the difference. I will remove it later once the other HPOs are updated.